test/integration/scheduler_perf/test-performance.sh

```diff
@@ -43,10 +45,10 @@ if ${RUN_BENCHMARK:-false}; then
   go test -c -o "perf.test"
 
   kube::log::status "performance test (benchmark) start"
-  "./perf.test" -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -test.short=false
+  "./perf.test" $glog_flags -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -test.short=false
```

Comment:
why not just set default flags in the perf test logic? this approach is going to be very brittle, for glog I can pass `-v 1` (space, two entries in argv, not `-v=1`) and this shell script parser is not going to handle it

Reply:
Any integration tests that run go binary or `go test` need to parse its glog flags. I think it is better to do this by a function, not inline into scheduler_perf.

for `-v 1(space)` not work, I will improve it if we agree that we need this function.

Reply:
> Any integration tests that run go binary or go test need to parse its glog flags. I think it is better to do this by a function, not inline into scheduler_perf.

why do they need to parse it? I disagree with this assertion. defaulting these flags can be done in the test code.

Reply:
Yes, we need it, especially we need it if we don't want to parse log flags in every go files.

Reply:
parsing it in every wrapper script is equivalent but more fragile. It's also actually more code and doesn't work when invoked directly.